### PR TITLE
[ Milestone 5 ] Performance

### PR DESCRIPTION
**Pull request by CakeInTech**
- Added the email column to the `owners` table.
- Added random data to the `owners` and the `visits` table.
- Used the `explain analyze` to check the execution time.

# Snapshots

### Before

image (3)

### After

image (5)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -62,3 +62,7 @@ CREATE TABLE visits (
     animals_id BIGINT REFERENCES animals (id),
     date_of_visit DATE NOT NULL
 );
+
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```
Comment (Davidosky007) [inline schema.sql]: * Kindly add indexes for FOREIGN KEYS or on columns frequently used in queries, this is one of the basic solutions to performance problems ( ie `CREATE INDEX index_name
ON table_name (column1, column2, ...);` ). Kindly apply this and see improvement in execution time 🙏 [Read More here](https://www.w3schools.com/sql/sql_create_index.asp)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @MohamedCK @Ashe546 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost git

## Highlights
* Well written PR description 👍 
* Good looking readme file 👍 
* Proper use of Gitflow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (CakeInTech) [reply schema.sql]: @Davidosky007  Thanks for the notice. 😄

**Comment by CakeInTech** [comment]
- Used `CREATE INDEX` to enhance the query execution time.
- Improved the execution time.
- ⛔ The `Before` snapshot is above.

## After Snapshot

image (5)

**Review by mukhammadrizooff — APPROVED** [review]
Hi @TEAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
- Good implementations ✔️ 

## Optional suggestions
- [x] [OPTIONAL] Good job so far! If you update the screenshot picture on the `AFTER` part to a [new one](https://github.com/MohamedCK/postgresql/pull/5#issuecomment-1359438781) it would be nice 👍🏼 

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**